src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -3409,6 +3425,53 @@ public void ScanAndProcessReturnValue (MethodBody methodBody)
 				}
 			}
 
+			public void ProcessAttributeDataflow (MethodDefinition method, Collection<CustomAttributeArgument> arguments)
+			{
```

Comment:
Should we check that the method is a .ctor?
The code will not work for this-calls correctly and it will ignore any potential return values.

Reply:
I don't think reflection stack would be able to do anything at runtime if this is not a constructor, so I'm not worried about ignoring return values.

Having an `explicitthis` constructor?... maybe. Added the handling for that just in case...